task2/rest-api-basics/gradlew.bat

```diff
@@ -0,0 +1,89 @@
+@rem
```

Comment:
@igor-zhuravlev move all extra files to git ignore

Reply:
@igor-zhuravlev also have some empty files like task2/rest-api-basics/profile-dev.gradle